docs/themes/thxvscode/layouts/partials/header.html

```diff
@@ -1,2 +1,4 @@
-<!-- EU Cookie Compliance JS -->
-<script src="//uhf.microsoft.com/mscc/statics/mscc-0.3.6.min.js"></script>
\ No newline at end of file
+<a role="button" id="skip-to-content" class="link-button" href="#main-content">Skip to content<i>&nbsp;</i><span
```

Comment:
i know this wasn't added by you, but I don't understand the need for `role="button"`. This is an anchor tag that navigates you to a different part of the page. [Role button](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) is often used when a div/span needs to be clickable.

```suggestion
<a id="skip-to-content" class="link-button" href="#main-content">Skip to content<i>&nbsp;</i><span
```

Reply:
ah, we're treating it as a button by using JS to turn a click into a focus on first focusable element 

![image](https://user-images.githubusercontent.com/1434956/95257456-30071400-07d9-11eb-9f0b-6ac3df4604b3.png)

still feels redundant, but at least I see the logic of why someone did that

Reply:
This was probably inherited from the template we were gifted. :) Worth filing a follow up issue?

Reply:
not really a big deal. not worth the cycles to track, just a nit since it's part of this PR